changelog.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Fixed
 - [[#3069] Fix Enums with String Values Return String Value for Enum Name](https://github.com/ballerina-platform/ballerina-standard-library/issues/3069)
 - [[#3067] Fix Single Quote Character Included in Field and Argument Names](https://github.com/ballerina-platform/ballerina-standard-library/issues/3067)
+- [[#3115] Fix Not Initializing Context per Service in GraphQL Subscriptions](https://github.com/ballerina-platform/ballerina-standard-library/issues/3115)
```

Comment:
This gives an entirely different message about the fix.

Not initializing Context per service is the way to go. There's nothing to fix in it. In fact, the issue is initializing context per service.